crates/bindings/src/table.rs

```diff
@@ -149,11 +149,6 @@ pub struct ScheduleDesc<'a> {
     pub scheduled_at_column: u16,
 }
 
-#[doc(hidden)]
-pub trait __MapRowTypeToTable {
```

Comment:
The downside of this removal is that there is now no way to link from row types to their table types in locally generated documentation.

We could consider adding a new trait:

```rust
/// Trait that documents which tables a row type is used for.
pub trait RowTypeForTable<T: Table> {}
```

implemented once for each table declared on a row type. This would serve no purpose aside from documentation.

Reply:
If we wanted to link from row types to tables in docs (which I would not take as a given that we do), I think we would do it by having the `table` attribute macro add a `doc = "Used as the row type for the table [this table handle type]"` attribute to the `struct` item. No trait is necessary.